coordinator/statement_executor.go

```diff
@@ -394,6 +385,12 @@ func (e *StatementExecutor) executeSetPasswordUserStatement(q *influxql.SetPassw
 }
 
 func (e *StatementExecutor) executeSelectStatement(stmt *influxql.SelectStatement, ctx *influxql.ExecutionContext) error {
+	// If there is a target and we are in a read only context, exit early since
+	// we will return an error.
+	if stmt.Target != nil && ctx.ReadOnly {
```

Comment:
Removed the commits that remove `IF NOT EXISTS` and `IF EXISTS` along with the commit that removes support for using write operations in a read only context. Modified the warning message so it doesn't mention 1.0.